.github/workflows/release.yaml

```diff
@@ -0,0 +1,81 @@
+---
+name: release
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  inspect:
+    runs-on: ubuntu-latest
+    container: golang:1.13
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set version
+        run: echo '::set-output name=version::${GITHUB_REF#refs/*/}'
+        id: set_version
+    outputs:
+      version: ${{ steps.set_version.outputs.version }}
+  release:
+    runs-on: ubuntu-latest
+    needs: inspect
+    if: ${{ needs.inspect.outputs.version }}
+    steps:
+      - name: Create the GitHub release.
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          tag_name: ${{ needs.inspect.outputs.version }}
+          release_name: ${{ needs.inspect.outputs.version }
+          draft: false
+          prerelease: false
+    outputs:
+      upload_url: ${{ steps.create_release.outputs.upload_url }}
```

Comment:
In the RHS, is `upload_url` implicitly defined as part of `create_release`?

Reply:
Yeah, `create_release` is a GitHub 1st party action. https://github.com/actions/create-release#outputs